spec/unit/capability_spec.rb

```diff
@@ -63,6 +63,34 @@
     expect(cns[:mappings]["host"]).to be_instance_of(Puppet::Parser::AST::PopsBridge::Expression)
   end
 
+  it "does not allow operator '+>' in a mapping" do
+    expect do
+    compile_to_catalog(<<-MANIFEST)
+      define test($hostname) {
+        notify { "hostname ${hostname}":}
+      }
+
+      Test consumes Cap {
+        host +> $hostname
+      }
+    MANIFEST
+    end.to raise_error(Puppet::ParseErrorWithIssue, /Illegal \+> operation on attribute/)
+  end
+
+  it "does not allow operator '*=>' in a mapping" do
+    expect do
+      compile_to_catalog(<<-MANIFEST)
+      define test($hostname) {
+        notify { "hostname ${hostname}":}
+      }
+
+      Test consumes Cap {
+        *=> $hostname
+      }
+      MANIFEST
+    end.to raise_error(Puppet::ParseError, /The operator '\* =>' in a Block Expression is not supported/)
```

Comment:
This does not seem right. I think the expression may be derived from BlockExpression and that a label is missing for it?